submissions/lalalilulu/dom-api-menu/script.js

```diff
@@ -0,0 +1,237 @@
+    },
+    {   type: "Окрашивание",
+        title: "Schwarzkopf Got2b Bright/Pastel Набор для тонирования волос",
+        description: "Набор для тонирования волос",
+        price: 299,
+        image: "./images/coloration/photo_2.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Magic Retouch",
+        description: "Краска для волос",
+        price: 204,
+        image: "./images/coloration/photo_3.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Colorista Spray",
+        description: "Тонирующий спрей для волос",
+        price: 299,
+        image: "./images/coloration/photo_4.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Crystal Clear",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_1.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pretzel Brown",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_2.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Wrapstar Hair-Rings Snake It Off",
+        description: "Резинка с лентой",
+        price: 1092,
+        image: "./images/accessories/photo_3.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pinkerbell",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_4.jpg"
+    }];
+
+function removeElementsIfExist(elementsList) {
+    if(elementsList.length > 0) {
+        Array.from(elementsList).forEach(element => element.remove());
+        return true;
+    }
+    return false;
+}
+
+function createElementWithClass(tagName, cssClass) {
+    const element = document.createElement(tagName);
+    element.classList.add(cssClass);
+    return element;
+}
+
+function createContentItem(product) {
+    const productElement = createElementWithClass("div", "content-item");
+
+    const itemImg = createElementWithClass("img", "content-item-image");
+    itemImg.setAttribute("src", product.image);
+    itemImg.setAttribute("alt", product.title);
+
+    const itemTitle = createElementWithClass("a", "content-item-title");
+    itemTitle.setAttribute("href", "#");
+    itemTitle.textContent = product.title;
+
+    const itemDescription = createElementWithClass("div", "content-item-description");
+    itemDescription.textContent = product.description;
+
+    const itemPrice = createElementWithClass("div", "content-item-price");
+    itemPrice.textContent = product.price;
+    const rubleSign = createElementWithClass("span", "rur");
+    rubleSign.textContent = "p";
+    itemPrice.appendChild(rubleSign);
+
+    productElement.append(itemImg, itemTitle, itemDescription, itemPrice);
+    return productElement;
+}
+
+
+document.getElementsByClassName("menu-list")[0].addEventListener("click", function (event) {
```

Comment:
You can destructure your event object here:
```suggestion
document.getElementsByClassName("menu-list")[0].addEventListener("click", function ({target}) {
```